app/queries/get_application_choices_for_providers.rb

```diff
@@ -12,40 +12,32 @@ class GetApplicationChoicesForProviders
   ].freeze
 
   def self.call(providers:, vendor_api: false, includes: DEFAULT_INCLUDES, recruitment_cycle_year: RecruitmentCycle.years_visible_to_providers)
-    providers = Array.wrap(providers).select(&:present?)
-
-    raise MissingProvider if providers.none?
+    raise MissingProvider if providers.blank? # super important!
```

Comment:
could you add a comment which explains why we need this (as discussed in person)?